.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @bajtos @juliangruber @pyropy @NikolasHaimerl
```

Comment:
Should we also deleted `reviewers` from [dependabot.yml](https://github.com/filecoin-station/spark-evaluate/blob/6007867d61b821453482691a8aa3e3c2fef1413b/.github/dependabot.yml#L12-L14)?